grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientFactoryProvider.java

```diff
@@ -18,10 +18,12 @@
 
 import com.linecorp.armeria.client.ClientFactory;
 import com.linecorp.armeria.client.ClientFactoryProvider;
+import com.linecorp.armeria.common.util.UnstableApi;
 
 /**
  * {@link ClientFactoryProvider} that creates a {@link GrpcClientFactory}.
  */
+@UnstableApi
```

Comment:
I wonder if it's ok for SPI implementations to be in `internal` package I guess so

Reply:
The `internal` packages are getting more and more crowded. Let me send a follow-up PR which 1) reorganizes the `internal` packages to have the same structure with non-internal packages (common, client, server) and 2) move the SPI impls there.